book/src/integrations/pam_and_nsswitch/suse.md

```diff
@@ -29,43 +29,36 @@ cp /etc/pam.d/common-session-pc  /etc/pam.d/common-session
 cp /etc/pam.d/common-password-pc /etc/pam.d/common-password
 ```
 
+> NOTE: Unlike other pam modules, replaces the functionality of `pam_unix` and can authenticate
+> local users securely.
+
 The content should look like:
 
 ```text
 # /etc/pam.d/common-account
 # Controls authorisation to this system (who may login)
-account    [default=1 ignore=ignore success=ok] pam_localuser.so
-account    sufficient    pam_unix.so
-account    [default=1 ignore=ignore success=ok]  pam_succeed_if.so uid >= 1000 quiet_success quiet_fail
 account    sufficient    pam_kanidm.so ignore_unknown_user
 account    required      pam_deny.so
 
 # /etc/pam.d/common-auth
 # Controls authentication to this system (verification of credentials)
 auth        required      pam_env.so
-auth        [default=1 ignore=ignore success=ok] pam_localuser.so
-auth        sufficient    pam_unix.so nullok try_first_pass
```

Comment:
I have tested this as working but I'm also happy to revert it so that pam_unix is a fall back.

Reply:
Yeah I think for now it is. I want to add the fallback code directly into Kanidm though so we're the "one stop" shop.